people_tool/static/people_tool/css/main.css

```diff
@@ -5,6 +5,12 @@
     pointer-events: none;
 }
 
+
+table.person-info-table {
```

Comment:
Ok sorry this is clearer now from what's below (but I'm curious why we don't do a combination of `.col-offset` and `col-*` like we do for the main table and for things like the search progress bar on other pages)?